src/cljs/nr/cardbrowser.cljs

```diff
@@ -273,6 +273,7 @@
         corp-factions ["Jinteki" "Haas-Bioroid" "NBN" "Weyland Consortium" "Neutral"]]
     (case side
       "All" (concat runner-factions corp-factions)
+      "Any Side" (concat runner-factions corp-factions)
```

Comment:
Does this need "Neutral" as well?

Reply:
No, Neutral is listed as one of the corp factions. We only need to add it if corp isn't present.